plugins/bulk-import/src/api/BulkImportBackendClient.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  ConfigApi,
+  createApiRef,
+  IdentityApi,
+} from '@backstage/core-plugin-api';
+
+import {
+  ImportJobResponse,
+  ImportJobStatus,
+  OrgAndRepoResponse,
+} from '../types/response-types';
+import { CreateImportJobRepository } from '../types/types';
+
+// @public
+export type BulkImportAPI = {
+  getRepositories: (page: number, size: number) => Promise<OrgAndRepoResponse>;
+  getRepositoriesFromOrg: (
+    orgName: string,
+    page: number,
+    size: number,
+  ) => Promise<OrgAndRepoResponse>;
+  getOrganizations: (page: number, size: number) => Promise<OrgAndRepoResponse>;
+  getImportJobs: (page: number, size: number) => Promise<ImportJobStatus[]>;
+  createImportJobs: (
+    importRepositories: CreateImportJobRepository[],
+    dryRun?: boolean,
+  ) => Promise<ImportJobResponse[]>;
+  checkImportStatus: (
+    repo: string,
+    defaultBranch: string,
+  ) => Promise<ImportJobStatus>;
+  removeRepository: (repo: string, defaultBranch: string) => Promise<Response>;
+};
+
+export type Options = {
+  configApi: ConfigApi;
+  identityApi: IdentityApi;
+};
+
+// @public
+export const bulkImportApiRef = createApiRef<BulkImportAPI>({
+  id: 'plugin.bulk-import.service',
+});
+
+export class BulkImportBackendClient implements BulkImportAPI {
+  // @ts-ignore
+  private readonly configApi: ConfigApi;
+  // private readonly identityApi: IdentityApi;
+
+  constructor(options: Options) {
+    this.configApi = options.configApi;
+    // this.identityApi = options.identityApi;
+  }
+
+  async getRepositories(page: number, size: number) {
+    // const { token: idToken } = await this.identityApi.getCredentials();
+
+    const backendUrl = this.configApi.getString('backend.baseUrl');
+    const jsonResponse = await fetch(
+      `${backendUrl}/api/bulk-import-backend/repositories?pagePerIntegration=${page}&sizePerIntegration=${size}`,
+      {
+        headers: {
+          // ...(idToken && { Authorization: `Bearer ${idToken}` }),
```

Comment:
If this is it not needed in the header, then we can get rid of the comment

Reply:
my [PR](https://github.com/janus-idp/backstage-plugins/pull/2034) that enables permission check in the bulk import frontend is ready, so removing the commented lines from here